drivers.json

```diff
@@ -1236,6 +1236,12 @@
                     "nvfbc_patch": true,
                     "driver_url": "http://international.download.nvidia.com/XFree86/Linux-x86_64/535.230.02/NVIDIA-Linux-x86_64-535.230.02.run"
                 },
+                {
+                    "version": "570.148.08",
+                    "nvenc_patch": true,
+                    "nvfbc_patch": true,
+                    "driver_url": ""
```

Comment:
can you add https://developer.download.nvidia.com/compute/cuda/repos/debian12/x86_64/cuda-drivers-570_570.148.08-1_amd64.deb as the driver url and re-run the readme generation script?

Reply:
done, sorry I couldn't find the link earlier.